lms/envs/common.py

```diff
@@ -2967,4 +2967,7 @@
 REDIRECT_CACHE_KEY_PREFIX = 'redirects'
 #####RACCOONGANG########
 DOC_LINK_BASE_URL = ''
+
+# JIVOSITE widget settings:
+JIVOSITE_WIDGET_ID = os.environ.get('JIVOSITE_WIDGET_ID')
```

Comment:
@wowkalucky 
it's correct, just add a default setting for JIVOSITE_WIDGET_ID variable.
 - for example: JIVOSITE_WIDGET_ID = os.environ.get('JIVOSITE_WIDGET_ID', '')
and prepare pull request into configuration repository
 - as an example: https://github.com/raccoongang/configuration/commit/51a12cd84874fd1d8d88461205a9f113466a729e